regression/cbmc-java/reachability-slice/test.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+A.class
+--reachability-slice --show-goto-functions --property "java::A.foo:(I)V.coverage.3" --cover location
```

Comment:
How about some assertion explicitly stated in the code rather than relying on coverage numbering? If you have to rely on coverage annotations, could you add a comment in the code to the block that you expect this to be mapped to?

Reply:
Assertion did not work here, surprisingly. The problem is that the assertion in Java throws an exception, so there's not really anything reachable after it, which makes the test vacuous. This is why I went for coverage criterion instead. I'll add a comment.

Reply:
Ok, thanks!